### DISCUSSION TITLE
Where is the Scroll Back To Top Button?

### DISCUSSION THREAD

**Discussion by yenny-dev** [forem/forem#20476] [Feature Requests]
I've just joined [dev.to](https://dev.to/) few hours ago and really enjoying reading [Meme Monday series](https://dev.to/search?q=Meme%20Monday) now.

However, everytime I trying to go to another post after reading one, 
I have to scroll way up to the top to click [last week's thread](https://dev.to/ben/meme-monday-33ej), 
and again in another post, again, again... (you can check in the capture pic)

![image](https://github.com/forem/forem/assets/106738220/5690ed08-7720-4d86-85f7-ad3596a580e4)

Since this way is so uncomfortable and inefficient, I felt a strong need for **`scroll to top button.`**
Does this function already exist but I can't find it?
![image](https://github.com/forem/forem/assets/106738220/5f196041-faaf-4eb3-815d-42ec2e04bad0)
I'm using Chrome browser and can't find any scroll back to top function when using [dev.to](https://dev.to/) website. 
Can't find any related mentions in FAQ, settings, search results..etc.

If there is no such function, I think this is very necessary element for better user experience. 
Expecting everyone's thoughts on this.

**Comment by lilxyzz** [comment]
I see where you are coming from with this one. On macOS Command + up arrow — Takes you back to the top of the page.

**Reply by yenny-dev** [reply]
OMG I didn't know that!!!
Thx for command information! Now it's all solved YEAH
